packages/xo-web/src/xo-app/home/index.js

```diff
@@ -456,6 +456,7 @@ const NoObjects = props =>
     mapValues(items, item => ({
       ...item,
       container: containers[item.$container || item.$pool],
+      usage: item.size > 0 ? item.physical_usage / item.size : 0,
```

Comment:
This should only be added on items of type `SR` and better choose another name to not override this property https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/xapi-object-to-xo.js#L515